packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 
-import * as ReactDOM from 'react-dom';
```

Comment:
Did you mean to remove this line?

Reply:
Yes.  I removed all references to ReactDOM.findDOMNode